### DISCUSSION TITLE
[combobox] Typing doesn't select new words when removing text

### DISCUSSION THREAD

**Discussion by erikkroes** [ing-bank/lion#1348] [General]
## Expected behavior

Typing in the textbox steers the dropdown. When I type "A", it should select the first word that starts with an "A". When I type "Ar", I would expect it to update the selection to "Artichoke" when I type the "r".

## Actual Behavior

Typing "A" selects the first word in both the dropdown and the textbox. Further typing happens -after- the selected word. So type "A", textbox becomes "Apple", type "r", textbox becomes "Appler".

**Comment by tlouisse** [comment]
Which browser etc? Can you make a gif that illustrates this?

**Comment by erikkroes** [comment]
![Oct-07-2020 09-52-43](https://user-images.githubusercontent.com/10405024/95302983-ed9a1180-0882-11eb-9c44-71f0b645d442.gif)

Using Safari

**Comment by tlouisse** [comment]
Yeah, good find. It's an issue in all browsers. Only happens indeed when you have a match on 'the way back'. (sets the activeIndex but not the checkedIdex)

**Comment by jorenbroekema** [comment]
I'm not sure that it should select the first match while typing, that's what active is for (so it focuses the first match). Selection should be a conscious user choice (pressing enter or click). This, to me, seems like intended behavior.

**Comment by tlouisse** [comment]
I think we do not inline autocomplete in textbox on purpose for best UX (we do on the way forward, not on the way backward), but we might want to do selection however... Will change label to discussion..
